content/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages.md

```diff
@@ -32,7 +32,9 @@ Then, make sure the _CNAME_ file is formatted correctly.
 
 If you have trouble pointing the default domain for your site to your custom domain, contact your DNS provider.
 
-You can also test whether your custom domain's DNS records are configured correctly. For more information, see "[Managing a custom domain for your {% data variables.product.prodname_pages %} site](/articles/managing-a-custom-domain-for-your-github-pages-site)."
+You can also test whether your custom domain's DNS records are configured correctly. There are multiple ways to query your DNS records:
+- Using the command line. For more information, see "[Managing a custom domain for your {% data variables.product.prodname_pages %} site](/articles/managing-a-custom-domain-for-your-github-pages-site)"; or
+- Using an [online DNS lookup tool](https://www.nslookup.io/).
```

Comment:
Hi @ruurtjan, thanks for this PR, and thanks for the disclosure!

We generally don't accept PRs that are self promoting, but I do agree that a mention of online DNS lookup tools would be helpful in this doc.

I don't think it's appropriate for our docs to be recommending a specific online DNS tool, so I think a generic mention would be more appropriate; a user can find and use whichever online tool they prefer. 

```suggestion
You can also use one of the following methods to test whether your custom domain's DNS records are configured correctly:

- A CLI tool such as `dig`. For more information, see "[Managing a custom domain for your {% data variables.product.prodname_pages %} site](/articles/managing-a-custom-domain-for-your-github-pages-site)".
- An online DNS lookup tool.
```